src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -88,6 +88,13 @@ message ConfigProto {
 
   repeated FixedResourceNameValueProto fixed_resource_name_values = 17;
 
+  // Set this value to True to enable String format functions for resource name
+  // entities. If this value is False, String format functions may or may not
+  // be enabled. This option is exposed for backward compatibility of Java clients.
+  // TODO(andrealin): Migrate this to a command-line option after migrating
+  // off artman.
+  bool enableStringFormatFunctionsOverride = 22;
```

Comment:
WDYT about making this a `BoolValue` instead of `bool`? That way, it can be true, false, or null - and can function as an override in both the true and false cases, instead of having the "may or may not" behaviour. I'm not sure how this works in the yaml files though.

Reply:
Sure, that makes sense. Done.